regparser/notice/build.py

```diff
@@ -341,6 +349,15 @@ def process_sxs(notice, notice_xml):
     notice['section_by_section'] = sxs
 
 
+def fetch_cfr_parts(notice_xml):
+    """ Sometimes we need to read the CFR part numbers from the notice
+        XML itself. This would need to happen when we've broken up a
+        multiple-effective-date notice that has multiple CFR parts that
+        may not be included in each date. """
```

Comment:
Yes, I should have written a test. That's been rectified!

This (and `notice_cfr_p` above) extracts the CFRs from the `<CFR/>` element in a notice. For, example:

```xml
<CFR>12 CFR Parts 1002, 1024, and 1026</CFR> 
```

So this gets us `['1002', '1024', '1026']`. 

This was to get around a problem where we [had split up a notice](https://github.com/willbarton/fr-notices/blob/fix-b/articles/xml/201/322/752-2.xml) [by date](https://github.com/willbarton/fr-notices/blob/fix-b/articles/xml/201/322/752-1.xml), but where Reg B had no changes for one of the dates.

So, in 2013-01384 (`201/322/752.xml`), we have [`752-1.xml`](](https://github.com/willbarton/fr-notices/blob/fix-b/articles/xml/201/322/752-1.xml) which just has changes for Z and X:

```xml
<CFR>12 CFR Parts 1002, 1024, and 1026</CFR> 
```
And [`752-2.xml`](](https://github.com/willbarton/fr-notices/blob/fix-b/articles/xml/201/322/752-2.xml), which has changes for B as well as Z and X:

```xml
<CFR>12 CFR Parts 1002, 1024, and 1026</CFR> 
```